### PR TITLE
Kinetic parameters added in the training reaction for the H_Abstraction family

### PR DESCRIPTION
**Pull request by hwpang**
The kinetic parameters in "Reaction kinetics of hydrogen abstraction from "polycyclic aromatic hydrocarbons by H atoms" calculated by Hou et al. at CCSD(T)/CBS or M06-2X/6-311g(d,p) have been added to the training reaction for the H_Abstraction family.

### CODE REVIEW

File: input/kinetics/families/H_Abstraction/training/reactions.py [hunk 1]
```diff
@@ -64121,3 +64121,187 @@
...
+    kinetics = Arrhenius(
+        A = (5.38e+09, 'cm^3/(mol*s)'),
+        n = 1.582,
+        Ea = (15.689, 'kcal/mol'),
+        T0 = (1, 'K'),
+    ),
+    reference = Article(
+        authors = ["'Dingyu Hou'", "'Xiaoqing You'"],
+        title = 'Reaction kinetics of hydrogen abstraction from polycyclic aromatic hydrocarbons by H atoms',
+        journal = "'Physical Chemistry Chemical Physics'",
```
Comment (mliu49) [inline input/kinetics/families/H_Abstraction/training/reactions.py]: Could you remove the extra quotes around these reference attributes?

**Comment by mliu49** [comment]
Sorry, could you remove the extra quotes for all of the training reactions which were added?

Also, could you expand on the long description a bit? I would suggest adding at least the following info:
- Geometry optimizations at B3LYP/6-311g(d,p) level
- Rate calculations using conventional TST, done with Multiwell 2016
- 1D Eckart tunneling corrections

Feel free to squash everything into one commit, and please adjust the commit message based on my other comment.

**Comment by mliu49** [comment]
@hwpang, I think this would be nice to include in the release if you have time to update it.

**Comment by hwpang** [comment]
New push rebased the changes and modified the long description per Max's comment.

**Comment by hwpang** [comment]
@xiaoruiDong New push changes (1) the description of all reactions to either `M06-2X/6-311g(d,p)//B3LYP/6-311g(d,p)` or `CCSD(T)/cc-pV(D,T)Z//B3LYP/6-311g(d,p)`, and (2) the rank of the first reaction to 3, as suggested offline. It also removes the redundant quotation marks and the duplicate index.

**Comment by xiaoruiDong** [comment]
@hwpang, looks good! I will merge it once checks were done.

**Review by xiaoruiDong — APPROVED** [review]

**Comment by xiaoruiDong** [comment]
It seems all added reactions have rmgpy.exceptions.UndeterminableKineticsError in the test.

e.g.
```
rmgpy.exceptions.UndeterminableKineticsError: (TemplateReaction(reactants=[Molecule(smiles="[H][H]"), Molecule(smiles="[c]1c2ccccc2cc2cc3ccccc3cc12")], products=[Molecule(smiles="c1ccc2cc3cc4ccccc4cc3cc2c1"), Molecule(smiles="[H]")]), 'Kinetics could not be determined. Unable to find matching template for reaction <Molecule "[H][H]"> + <Molecule "[c]1c2ccccc2cc2cc3ccccc3cc12"> <=> <Molecule "c1ccc2cc3cc4ccccc4cc3cc2c1"> + <Molecule "[H]"> in reaction family <KineticsGroups "H_Abstraction/groups">.Trying to match [<Entry index=0 label="X_H_or_Xrad_H_Xbirad_H_Xtrirad_H">, <Entry index=1 label="Y_rad_birad_trirad_quadrad">] but matched []')
```

This is slightly irrelevant: some bugs in the rules.py and groups.py. of H abstraction. Do you mind taking a look at it?

**Comment by hwpang** [comment]
@xiaoruiDong Thanks for the review! I think the test failed because 2019 me labeled the reaction site wrong... This new push should fix the problem.

**Comment by hwpang** [comment]
@xiaoruiDong The new push fixed the error. The continuous test failed with some cantera errors that are not related to this PR.

**Comment by xiaoruiDong** [comment]
> @xiaoruiDong The new push fixed the error. The continuous test failed with some cantera errors that are not related to this PR.

Looks good！

**Pull request merged**